src/shared/directives/conditional-show/conditional-show.directive.ts

```diff
@@ -1,213 +0,0 @@
-import { AfterViewInit, Directive, ElementRef, Input, OnDestroy, Renderer2 } from '@angular/core';
```

Comment:
Why has this been removed as part of this PR? Wasn't it removed in a previous PR? This feels like a risky thing to do as part of this PR as it's not absolutely necessary... or is it? 😬